### PR TITLE
Add optional markup to the basic menu item

### PR DESCRIPTION
**Pull request by tari01**

### CODE REVIEW

File: src/idobasicmenuitem.c [hunk 1]
```diff
@@ -359,6 +359,12 @@ ido_basic_menu_item_new_from_model (GMenuItem    * menu_item,
 
   item = ido_basic_menu_item_new ();
 
+    IdoBasicMenuItemPrivate *p = ido_basic_menu_item_get_instance_private(IDO_BASIC_MENU_ITEM(item));
+    gboolean bUseMarkup = FALSE;
```
Comment (sunweaver) [inline src/idobasicmenuitem.c:363]: The variable style seems to be different from what developers have used in the rest of the code (lower_case_with_underscores). We don't want to mix styles, do we?

**Review by sunweaver — COMMENTED** [review]
We need to "talk" about coding style and variable naming. For existing projects, I normally immitate the style already in use. Please get back to me on that.

File: src/idobasicmenuitem.c (hunk 1 above)
Comment (tari01) [reply src/idobasicmenuitem.c]: > The variable style seems to be different from what developers have used in the rest of the code (lower_case_with_underscores). We don't want to mix styles, do we?

Converted in https://github.com/AyatanaIndicators/ayatana-ido/pull/17/commits/b7b61e345a19cacb937cf0fb14398545ae8b9349

**Review by sunweaver — APPROVED** [review]
LGTM now.

**Pull request merged**